checkQC/default_config/config.yaml

```diff
@@ -8,11 +8,15 @@ parser_configurations:
     # Path to where the bcl2fastq output (i.e. fastq files, etc) is located relative to
     # the runfolder
     bcl2fastq_output_path: Data/Intensities/BaseCalls
+  SamplesheetParser:
+    samplesheet_name: SampleSheet.csv
 
 default_handlers:
     - name: UndeterminedPercentageHandler
       warning: unknown
       error: 10
+    - name: UnidentifiedIndexHandler
+      significance_threshold: 0.001 # fraction of reads
```

Comment:
Shouldn't this be 0.01? If we're using the same fraction as in Sisyphus, which is based on QC criteria documents used internally at SNP&SEQ.

Reply:
Yeah. This is just a test level which I added to make sure I got something on the folders I tested on.